CMakeLists.txt

```diff
@@ -282,6 +282,9 @@ mark_as_superbuild(Slicer_BUILD_QT_DESIGNER_PLUGINS)
 option(Slicer_BUILD_EXTENSIONMANAGER_SUPPORT "Build Slicer extensions manager" ON)
 mark_as_superbuild(Slicer_BUILD_EXTENSIONMANAGER_SUPPORT)
 
+option(Slicer_BUILD_APPLICATIONUPDATE_SUPPORT "Build Slicer with application update support" ON)
```

Comment:
@lassoan Should this be set to OFF by default when going through the SlicerCustomAppTemplate process as a Slicer custom app shouldn’t be checking for Slicer application updates?

Reply:
> Should this be set to OFF by default when going through the SlicerCustomAppTemplate process 

Indeed, default can be set to off if the main application name is different from Slicer.

((I will be back in the office next week and should be able to further comment.))

Reply:
Yes, custom applications could either disable application updates or set a different default update server URL.

Reply:
https://github.com/KitwareMedical/SlicerCustomAppTemplate/pull/43 issued for this